ancar/portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
```

Comment:
this file is not mentiond in the desc...

Reply:
So sorry! Apparently it wasn't tracked and I didn't notice and added it now. 
Room for improvement :(

Reply:
It was from the previous pull request

Reply:
so do you want to revert it, or add it to the description? alternatively create a ranch with only this, have an other pr, and rebase this pr, so it does not show it anymore, but the file is in the master.. ;)

Reply:
I want to add it to the description.